src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,7 +14,9 @@
     private Phone phone;
     private Email email;
     private Address address;
-
+    private int sequenceNumber;
+    public int nextSequenceNumber;
```

Comment:
a couple of things:
- you need to make it `static` to make it a class level member
- this need not be public; in general try to start off with private and then change to protected or public only if it is absolutely necessary